cra-server/router/routes.js

```diff
@@ -11,23 +11,25 @@ const rateLimit = require("express-rate-limit");
 const getAxiosInstance = require("../functions/getAxiosInstance");
 const validateURL = require("../validations/validateURL");
 const validateAdminURL = require("../validations/validateAdminURL");
+const getSecurityInfoFromEnv = require("../functions/getSecurityInfoFromEnv");
 
 // required for codeQL to ensure that logins are rate limitted
 const loginLimiter = rateLimit({
   windowMs: 15 * 60 * 1000, // 15 minutes
   max: 100, // limit each IP to 100 requests per windowMs
 });
+const security = getSecurityInfoFromEnv();
 
 // used for client authentication (so we can trust the server)
-const keystore = fs.readFileSync(
-  path.join(__dirname, "../../") + "ssl/keystore.p12"
+const pfx = fs.readFileSync(
+  path.join(__dirname, "../../") + security.pfx
 );
 // server for server authentication (so the server can trust us)
-const truststore = fs.readFileSync(
-  path.join(__dirname, "../../") + "ssl/truststore.p12"
+const ca = fs.readFileSync(
+  path.join(__dirname, "../../") + security.ca
 );
 
-passphrase = "egeria";
+passphrase= security.pfx_passphrase;
```

Comment:
so here you are using passphrase environment variable as a VALUE. 
Either is fine but should be consistent.

Reply:
.. and more occurrences in file later on

Reply:
@planetf1 SorryI don't understand. Is there something inconsistent?

Reply:
Yes, on line 25/26 I see:
```
# This is the location of the file containing the key used to secure communications between the browser and the presentation server
EGERIA_PRESENTATIONSERVER_EGERIA_BROWSER_PASSPHRASE=ssl/keys/server.key
```
So I got the impression there the value was a file - whereas here the passphrase is referred to as a value?